src/utils/utils_windows_common.c

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#include <windows.h>
+
+#include <processenv.h>
+
+#define BUFFER_SIZE 1024
+
+int util_env_var(const char *envvar, char *buffer, size_t buffer_size) {
+    return GetEnvironmentVariableA(envvar, buffer, (DWORD)buffer_size);
+}
+
+int util_env_var_has_str(const char *envvar, const char *str) {
+    char buffer[BUFFER_SIZE];
+    int rc = util_env_var(envvar, buffer, BUFFER_SIZE);
+    if (0 < rc && rc < BUFFER_SIZE) {
+        return strstr(buffer, str) ? 1 : 0;
```

Comment:
IMHO using `strstr(...) != NULL ` would make this code easier to read.

Reply:
Done